portfolio/src/main/webapp/map.js

```diff
@@ -1,58 +1,76 @@
-class MapHandler {
-  // The coordinates for all markers and their related properties.
-  static MAP_CENTER = {
-    lat: 43,
-    lng: 169,
-  };
-
-  static SUZHOU_COORD = {
-    lat: 31.293463,
-    lng: 120.666522,
-    title: 'My Hometown',
-    id: 'suzhou',
-  };
+/** 
+ *  The coordinate class which registers information for 
+ *  each marker. 
+ */
+class Coordinate {
+  
+  constructor(latitude, longitude, title, id) {
+    /** @member {float} latitude */
```

Comment:
Make these members private and have getter methods for access. Also if they never change, mark them as constants.

Reply:
thanks! I don't know about all these rules before. We can discuss it tomorrow.